src/_utils/focusVisibleProvider/index.tsx

```diff
@@ -17,16 +17,29 @@ const pointerMoveEventList = [
 
 const pointerDownEventList = ['mousedown', 'pointerdown', 'touchstart']
 
+export const FOCUS_VISIBLE_CSS_CLASS = 'focus-visible'
+
 type FocusVisibleProviderProps = {
   children: ReactNode
+  hasGlobalClassName?: boolean
 }
 
 export const FocusVisibleContext = createContext(false)
 
-export const FocusVisibleProvider = ({ children }: FocusVisibleProviderProps) => {
+export const FocusVisibleProvider = ({
+  children,
+  hasGlobalClassName,
+}: FocusVisibleProviderProps) => {
   /* When the provider first loads, assume the user is in pointer modality. */
   const [hadKeyboardEvent, setHadKeyboardEvent] = useState(false)
 
+  if (hasGlobalClassName) {
+    useEffect(() => {
+      // Provide a global css selector for legacy components
```

Comment:
Move this comment to the props itself to make it more discoverable.
// Add a 'focus-visible' CSS class on the < body > element.

Reply:
Done